examples/kyb-app/package.json

```diff
@@ -24,13 +24,15 @@
     "dayjs": "^1.11.6",
     "form-data-encoder": "^3.0.0",
     "i18n-iso-countries": "^7.6.0",
+    "jsonschema": "^1.4.1",
```

Comment:
do you use `jsonschema`?

Reply:
Im not using but after adding `tailwind-merge` it started to complain that json-schema is missing and broke build,installation of `json-schema` resolved issue.

Reply:
I think it worth extra check because t-merge doesnt have any dependency
<img width="763" alt="image" src="https://github.com/ballerine-io/ballerine/assets/6435752/6135d6fa-c042-4af3-9df4-7b8f80626e03">

Reply:
@liorzam you are right,it seems issue was in different place.Anyways code is updated and now `ctw` from `ui` is used instead.